contracts/UpkeepRegistrationRequests.sol

```diff
@@ -122,6 +127,13 @@ contract UpkeepRegistrationRequests is Owned {
                 amount,
                 hash
             );
+        } else {
+            uint96 newBalance = s_pendingRequests[hash].balance + amount;
+            require(newBalance >= amount, "overflow");
```

Comment:
```suggestion
            require(newBalance >= amount, "overflow");
```
If it is equal, that means the amount was zero, not an overflow. I'd recommend using SafeMath96 here instead, it's already in the repo.